mcr_navigation/mcr_direct_base_controller/ros/launch/direct_base_controller_standalone.launch

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
```

Comment:
I think you can remove the standalone version if it's the same as `direct_base_controller.launch`